lang/en.json

```diff
@@ -182,5 +242,35 @@
 	"CYPHERSYSTEM.Platinum": "Platinum",
 	"CYPHERSYSTEM.Gold": "Gold",
 	"CYPHERSYSTEM.Silver": "Silver",
-	"CYPHERSYSTEM.Copper": "Copper"
-}
+	"CYPHERSYSTEM.Copper": "Copper",
+	
+	"CYPHERSYSTEM.AlreadyHasThisItem": "already has this item.",
+	"CYPHERSYSTEM.CannotMoveNotOwnedItem": "You can’t move items you don’t have.",
+	"CYPHERSYSTEM.CanOnlyMoveCertainAmountOfItems": "You can only move between 1 and ",
+	"CYPHERSYSTEM.CanOnlyCreateMacroForOwnedItems": "You can only create macros for owned Items.",
+	"CYPHERSYSTEM.MacroOnlyAppliesToPC": "This macro only applies to PCs.",
+	"CYPHERSYSTEM.DebilitatedPCEffort": "Debilitated PCs can’t spend Effort.",
+	"CYPHERSYSTEM.NotEnoughMight": "You don’t have enough Might points.",
+	"CYPHERSYSTEM.NotEnoughSpeed": "You don’t have enough Speed points.",
+	"CYPHERSYSTEM.NotEnoughIntellect": "You don’t have enough Intellect points.",
+	"CYPHERSYSTEM.SpendTooMuchEffort": "You tried to apply more Effort than your Effort score allows.",
+	"CYPHERSYSTEM.NotEnoughPoint": "You don’t have enough points in",
```

Comment:
This isn’t elegant. Needs a better solution. Can translations be nested?

Reply:
Unfortunately, I have not found I way to do so outside HTML.
I mean this works in HTML:
`"ACTOR.pool.additional.decrease": "Subract 1 {{#if (eq data.additionalPool.additionalTeenPoolName '')}}Luck{{else}}{{data.additionalPool.additionalTeenPoolName}}{{/if}} point (ctrl/cmd-click to subract 10 points)"`

But this does not work in JS:
`"CYPHERSYSTEM.NotEnoughPoint": "You don’t have enough points in ${pool}."`
![image](https://user-images.githubusercontent.com/38940602/112118720-f0808880-8bbc-11eb-9743-9aded61fbf80.png)

Reply:
Same if you put the little ` inside the localisation string or if you put the localisation string inside a `${}`.
First example:
`"CYPHERSYSTEM.NotEnoughPoint": "`You don’t have enough points in ${pool}.`"`
![image](https://user-images.githubusercontent.com/38940602/112119014-405f4f80-8bbd-11eb-8a68-00c2e790eae6.png)

Second example:
`return ui.notifications.notify(`${game.i18n.localize("CYPHERSYSTEM.NotEnoughPoint")}`);`
![image](https://user-images.githubusercontent.com/38940602/112119536-c11e4b80-8bbd-11eb-9c28-b2fe394705d9.png)

Combination of both:
`"CYPHERSYSTEM.NotEnoughPoint": "`You don’t have enough points in ${pool}.`"`
`return ui.notifications.notify(`${game.i18n.localize("CYPHERSYSTEM.NotEnoughPoint")}`);`
![image](https://user-images.githubusercontent.com/38940602/112119349-959b6100-8bbd-11eb-9a97-1fdae13dc014.png)

Reply:
So either I am missing something or it is not possible.

Reply:
I’ll take a look at other systems and see how they solved this.

Reply:
It is my bad...
I definitely missed that freaking handsome piece of method:
![image](https://user-images.githubusercontent.com/38940602/112154572-b1fec400-8be4-11eb-97ad-07308eb525ac.png)

Reply:
Nobodies fault. I overlooked that as well until I found the example in dnd5e.

Reply:
Refactored translation